### PR TITLE
Revised the devdocs contribution guide

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR):

- Addresses editorial issues 
- Adds info about the review process
- Adds info about completing issue/PR templates
- Adds info about creating an issue prior to making global changes
- Adds info about directly contacting staff and maintainers

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 1. Review the [DevDocs guidelines](#contribution-guidelines).
 
 ## Contribute docs
 
+The following diagram shows the contribution workflow:
+
 ![Contributing workflow](https://devdocs.magento.com/common/images/contribute-write-submit-pr.png)
 
-1. Create a new branch on your fork. Use a name that best describes the work, or references a GitHub issue number.
+**Tip!** If you are not sure where to start contributing, search for issues with the [`help wanted`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [`good first issue`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
```
Comment (jcalcaben) [inline .github/CONTRIBUTING.md]: We should also say that PR's linked to issues with the `help wanted` tag automatically have a high priority for processing.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
+1. Create a pull request to the [magento/devdocs](https://github.com/magento/devdocs) repository.
 
-In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, such as `develop`.
+   In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, such as `develop`.
 
-Provide links to devdocs for the files you are changing. This is not required if the list is long.
+1. Complete the pull request template.
 
-If you are updating an example from source code, include a link to the file in the repository.
+   We will close your pull request if you fail to complete the template.
```
Comment (jcalcaben) [inline .github/CONTRIBUTING.md]:
```suggestion
   **We will close your pull request if you fail to complete the template.**
```
Feel like this should be boldly stated.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 
-The DevDocs team and Maintainers will review the PR and help with formatting and navigation.
+1. After creating a pull request, a DevDocs staff member or maintainer will review it and may ask you to make revisions.
```
Comment (jcalcaben) [inline .github/CONTRIBUTING.md]: Should we add a note that a lack of response in a reasonable amount of time will result in a rejected PR?

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (hguthrie) [reply .github/CONTRIBUTING.md]: Be ready to follow through on this. 🙂

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 
-1. Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
-1. For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
-1. [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
-1. Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+- Write content using markdown. See the [Templates](#templates) section for examples.
+- Review existing [pull requests](https://github.com/magento/devdocs/pulls) and [issues](https://github.com/magento/devdocs/issues) to avoid duplicating work.
+- For large contributions, or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This helps prevent duplicate or unnecessary work.
+- Do not make global find/replace type changes without first creating and issue and discussing it with us. Global changes can have unintended consequences.
```
Comment (hguthrie) [inline .github/CONTRIBUTING.md]:
```suggestion
- Do not make global find-and-replace changes without first creating an issue and discussing it with us. Global changes can have unintended consequences.
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
+- Review your work for basic typos, formatting errors, or ambiguous sentences before opening a pull request.
+- Revise pull requests according to review feedback. We will close pull requests that require an inordinate amount of time to review and process (especially for minor changes) if you fail to make revisions according to review feedback.
+- Do not directly contact DevDocs team members or maintainers on Slack to review your pull request unless it has been open for more than five days. We have a process and queue for pull requests that everyone must follow.
 
-Focus on the content with useful information, code samples, and important notes for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+## Tips for writing content
 
 Use the following guidelines to help you with the writing process:
 
+- Focus your efforts on providing useful information for your fellow Magento developers and community members. For example, new or revised code samples, important notes, clarification of vague or ambiguous content.
```
Comment (hguthrie) [inline .github/CONTRIBUTING.md]:
```suggestion
- Focus your efforts on providing useful information for your fellow Magento developers and community members. For example, consider providing or revising code samples, important notes, and clarifying vague or ambiguous content.
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -107,17 +112,18 @@ title: Continue with your installation
...
-Enter as much information as you can including content corrections, steps to reproduce, command or code updates, or questions for clarifications.
+If you find a typo or errors in Magento DevDocs, you can either fix it with a pull request (as described above) or you can report it by creating an issue in the DevDocs GitHub repository.
+
+You must complete the issue template. We will close your issue if you fail to complete the template. Enter as much information as you can, including content corrections, steps to reproduce, command or code updates, or questions for clarifications.
 
-**Note:** Check the [existing issues](https://github.com/magento/devdocs/issues) on GitHub to see if someone has already reported the issue.
+**Note:** Check the existing [issues](https://github.com/magento/devdocs/issues) on GitHub to see if someone has already reported the issue.
 
-You have a couple of options to enter an issue:
+You have a couple of options to create an issue:
```
Comment (hguthrie) [inline .github/CONTRIBUTING.md]:
```suggestion
You can provide feedback using one of the following options:
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
+As you contribute PRs, you gain [Contribution Points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#points).
 
 If you write and contribute a full topic, we will add your name (or your company's name) at the top of the DevDocs page and link it to your blog or website!
 
 ## Get started
 
 ![Get started workflow](https://devdocs.magento.com/common/images/contribute-prerequisites.png)
 
-1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) to your account. Partners are required to add 2FA protection when contributing to Magento repositories.
+1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) enabled on your account. We require partners to add 2FA protection when contributing to Magento repositories.
```
Comment (dshevtsov) [inline .github/CONTRIBUTING.md]: This is a note for a particular audience and can be skipped by general audience, so it should be represented as a separate note for partners. For example: _**Note for partners**: Add 2FA protection when contributing to Magento repositories._
Also, do we have a description of a contributor/maintainer/partner/etc roles somewhere? How do I know what role should I be in when reading the guidelines?

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 1. Sign the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement).
 1. [Fork](https://help.github.com/articles/fork-a-repo/) the [DevDocs repository](https://github.com/magento/devdocs). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
 1. Review the [DevDocs guidelines](#contribution-guidelines).
 
 ## Contribute docs
```
Comment (dshevtsov) [inline .github/CONTRIBUTING.md]: This is not part of the PR, but I'd like to highlight that we should avoid use of abbreviations: _docs_ -> _documentation_

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (dshevtsov) [reply .github/CONTRIBUTING.md]: There are two different names for the same entity in this topic:
1. Magento's tech writer team
2. DevDocs staff

It is not clear whether they are the same or different. We should use one name for one entity to avoid ambiguity.

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
-
-You can update existing or add new topics in their respective Magento 2 versioned directories (2.1, 2.2, 2.3, and onward). If you need help finding a directory for your content, we can help in your PR.
-
 The following guidelines may answer most of your questions and help you get started:
 
-1. Check [existing pull requests](https://github.com/magento/devdocs/pulls) and make sure you are not duplicating work!
-1. For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
-1. [Chunk many small/medium changes](#tips-for-writing-content) into one or two PRs. This helps us to efficiently and effectively facilitate your contribution.
-1. Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+- Write content using markdown. See the [Templates](#templates) section for examples.
```
Comment (dshevtsov) [inline .github/CONTRIBUTING.md]:
```suggestion
- Write content using Markdown. See the [Templates](#templates) section for examples.
```

Comment (dshevtsov) [reply .github/CONTRIBUTING.md]: Our templates do not comply with pure Markdown markup. It makes sense to add information that we use [kramdown](https://kramdown.gettalong.org/) that implement several Markdown extensions.

Comment (dshevtsov) [reply .github/CONTRIBUTING.md]: The _Templates_ section contains links to [Markdown](https://github.com/magento/devdocs/blob/master/guides/v2.1/contributor-guide/templates/basic_template.md) files. I'm not sure that this is a good idea. For example, if I'll follow the provided guidelines how to add notes,
```
{%raw%}
{%
include note.html
type='info'
content='This is a note callout. You can use these to provide important information on a topic.'
%}
{%endraw%}
```

I won't see the expected note.

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 - Use notes to alert readers about important details.
-- Use cross-references to other topics if appropriate. We can help you with the syntax if it is not clear. The template provides an example you can use.
+- Use cross-references to other topics sparingly. We can help you with the syntax if it is not clear. The template provides an example you can use.
 - Need help with markdown? See our [templates](#templates).
 
-### Preview your work on local
+### Review changes locally
 
-Optional. To preview your changes in HTML output, follow the instructions in the [README](https://github.com/magento/devdocs/blob/master/README.md) to build the devdocs site locally using Jekyll.
+_(Optional)_ To review your changes in HTML output, follow the instructions in the [README](https://github.com/magento/devdocs/blob/master/README.md) to build the devdocs site locally using Jekyll.
```
Comment (dshevtsov) [inline .github/CONTRIBUTING.md]: Not sure if will work. The README does not contain any instructions about how to build and preview subrepositories.

File: .github/CONTRIBUTING.md [hunk 11]
```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 
 ### Templates
 
-We provide templates to help you started writing new content and understanding markdown formatting:
+We provide templates to help you get started writing new content and understanding markdown formatting:
```
Comment (dshevtsov) [inline .github/CONTRIBUTING.md]:
```suggestion
We provide templates to help you get started writing new content and understanding Markdown formatting:
```

File: .github/CONTRIBUTING.md (hunk 7 above)
Comment (jeff-matthews) [reply .github/CONTRIBUTING.md]: Good question. This is for contributors only.

I wrote some maintainer guidelines a few weeks ago after onboarding a couple of new maintainers. I shared with them directly, but I've been meaning to make them public. I sent @lorikrell a copy a few days ago so she could add to the repo wiki, but she's been at Imagine all week. Instead of adding them to the wiki, maybe we should add to the repo as `.github/MAINTAINER-GUIDELINES.md`.

Comment (jeff-matthews) [reply .github/CONTRIBUTING.md]: See https://github.com/magento/devdocs/pull/4535

File: .github/CONTRIBUTING.md (hunk 8 above)
Comment (hguthrie) [reply .github/CONTRIBUTING.md]: I second this.

File: .github/CONTRIBUTING.md (hunk 10 above)
Comment (jeff-matthews) [reply .github/CONTRIBUTING.md]: Contributors may not be able to build some repos even after we update the README since some repos are private.

File: .github/CONTRIBUTING.md [hunk 12]
```diff
@@ -4,87 +4,107 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 ## Get started
 
 ![Get started workflow](https://devdocs.magento.com/common/images/contribute-prerequisites.png)
 
-1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) to your account. Partners are required to add 2FA protection when contributing to Magento repositories.
+1. Make sure you have a [GitHub account](https://github.com/signup/free).
+
+    **Note for partners:** Add [2FA](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
+
 1. Sign the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement).
```
Comment (lorikrell) [inline .github/CONTRIBUTING.md]: One big point of feedback from first time contributions: It was confusing to have CLA signing in these steps. It was  better to have in the PR creation and submission steps. I can give you an updated image to match if you want to move it. I have the source for it.

File: .github/CONTRIBUTING.md [hunk 13]
```diff
@@ -4,87 +4,107 @@ Share your knowledge with the community by contributing to Magento DevDocs!
...
 
-## Contribute docs
+## Contribute documentation
+
+The following diagram shows the contribution workflow:
 
 ![Contributing workflow](https://devdocs.magento.com/common/images/contribute-write-submit-pr.png)
 
-1. Create a new branch on your fork. Use a name that best describes the work, or references a GitHub issue number.
+**Tip!** If you are not sure where to start contributing, search for issues with the [`help wanted`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [`good first issue`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels. These issues receive higher priority for processing.
```
Comment (lorikrell) [inline .github/CONTRIBUTING.md]: Would it be worth noting the groomed label issues have been reviewed by DevDocs and available for working?

File: .github/CONTRIBUTING.md (hunk 12 above)
Comment (jeff-matthews) [reply .github/CONTRIBUTING.md]: Thanks Lori. I removed it from this section. It already exists in the Create a PR section. No need for an image.

File: .github/CONTRIBUTING.md (hunk 13 above)
Comment (jeff-matthews) [reply .github/CONTRIBUTING.md]: I don't think it's necessary to mention that we've reviewed them, but I'll add `groomed` to the list of labels.

**Comment by jeff-matthews** [comment]
running tests
